examples/ratelimtit/Dockerfile-service

```diff
@@ -0,0 +1 @@
+FROM nginx:latest
```

Comment:
im not entirely comfortable with this, the flask container would possibly work here 

if we are going to use `nginx` then there is no need for a dockerfile - it can just be specified in the docker-compose file

Reply:
Done.